app/Policies/WorkerPolicy.php

```
@@ -36,6 +36,13 @@ public function create(User $user, Server $server, ?Site $site = null): bool
             $server->processManager();
     }
 
+    public function manage(User $user, Server $server, ?Site $site = null): bool
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$site'. (undefined)

(UnusedFormalParameter)